### PR TITLE
added files

### PR DESCRIPTION
**Pull request by faikp**

### CODE REVIEW

**Review by mujeeb100 — APPROVED** [review]

File: style.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    backdrop-filter: blur(5px);
+
+    overflow: hidden;
+
+    display: flex;
+    align-items: center;
+
+    border-top: 1px solid rgba(255, 255, 255, 0.5);
+    border-left: 1px solid rgba(255, 255, 255, 0.5);
+}
```
Comment (Rehan-Khan-Engineer) [inline style.css:78]: How about using shorthand property?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="card">
+            <div class="content">
+                <h2>01</h2>
+                <h3>Card One</h3>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quisquam labore repellendus illo, deleniti veniam doloribus laudantium  vitae!</p>
+                <a href="#">Read More</a>
```
Comment (Rehan-Khan-Engineer) [inline index.html:17]: can add hover effect to Read more. It might look better!

**Comment by OSAMA478** [comment]
superb.....

**Review by OSAMA478 — APPROVED** [review]

**Review by anask601 — APPROVED** [review]
LGTM

**Pull request merged**